### PR TITLE
Docs: Move section about log-level to new troubleshooting page.

### PR DESCRIPTION
**Pull request by csadorf**

### CODE REVIEW

File: docs/source/intro/troubleshooting.rst [hunk 1]
```diff
@@ -275,3 +275,49 @@ Finally, start a Jupyter server:
    $ jupyter notebook
 
 and from the newly opened browser tab select ``New -> <aiida.kernel.name>``
+
+Increasing the debug level
```
Comment (sphuber) [inline docs/source/intro/troubleshooting.rst:279]:
```suggestion
Changing the logging level
```

File: docs/source/intro/troubleshooting.rst [hunk 2]
```diff
@@ -275,3 +275,49 @@ Finally, start a Jupyter server:
...
+  verdi config logging.circus_loglevel DEBUG
+
+
+For each profile that runs a daemon, there will be two unique logfiles, one for
+AiiDA log messages (named ``aiida-<profile_name>.log``) and one for the circus logs (named ``circus-<profile_name>.log``). Those files can be found
+in the ``~/.aiida/daemon/log`` folder.
+
+After rebooting the daemon (``verdi daemon restart``), the number of messages
+logged will increase significantly and may help in understanding
+the source of the problem.
```
Comment (sphuber) [inline docs/source/intro/troubleshooting.rst:302]:
```suggestion
After restarting the daemon (``verdi daemon restart``) additional messages will be logged that may help in understanding the source of the problem.
```

**Comment by sphuber** [comment]
@csadorf I think you may have accidentally force pushed the wrong branch here.

**Comment by csadorf** [comment]
> @csadorf I think you may have accidentally force pushed the wrong branch here.

I'll fix that.

**Comment by csadorf** [comment]
> > @csadorf I think you may have accidentally force pushed the wrong branch here.
> 
> I'll fix that.

@sphuber Done.

File: docs/source/intro/troubleshooting.rst [hunk 3]
```diff
@@ -275,3 +275,51 @@ Finally, start a Jupyter server:
...
 and from the newly opened browser tab select ``New -> <aiida.kernel.name>``
+
+.. _intro:increase-logging-verbosity:
+
+Increasing the logging verbosity
+--------------------------------
+
+By default, the logging level of AiiDA is minimal to avoid filling logfiles.
+Only warnings and errors are logged to the daemon log files, while info and debug
+messages are discarded.
```
Comment (sphuber) [inline docs/source/intro/troubleshooting.rst:286]:
```suggestion
Only warnings and errors are logged to the daemon log files, while info and debug messages are discarded.
```
Should just apply style requirements everywhere:
 * one sentence per line
 * proper `.. code-block:: console` for `verdi` commands ..
 * .. plus prefix them with a `$` sign

Comment (csadorf) [reply docs/source/intro/troubleshooting.rst:286]: On it.

**Pull request merged**